### PR TITLE
Add cleanup script running at night on botmaster

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by coolo** [comment]
This is mainly to archive it - there is no deployment from this

File: gocd/admin/cleanup.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            """delete from modifiedfiles where modificationid in
+               (select id from modifications where modifiedtime < current_date - interval '10' day)"""
+        )
+        # cur.execute("delete from modifications where modifiedtime < current_date - interval '10' day")
+    conn.close()
+
+
+def main():
+    # pull new image - if registry is down, better stop here
+    subprocess.run(
```
Comment (Vogtinator) [inline gocd/admin/cleanup.py:127]: Can you tag the old image?

Comment (coolo) [reply gocd/admin/cleanup.py:127]: I can't, but I put the script in git so you can :)

Comment (Vogtinator) [reply gocd/admin/cleanup.py:127]: I can't test it, what about this:

```
   subprocess.run(
        [
            "docker",
            "tag",
            "registry.opensuse.org/opensuse/tools/images/containers_tumbleweed/gocd-agent-release-tools:latest",
            "registry.opensuse.org/opensuse/tools/images/containers_tumbleweed/gocd-agent-release-tools:previous",
        ],
        check=True,
    )

   subprocess.run(
        [
            "docker",
            "pull",
            "registry.opensuse.org/opensuse/tools/images/containers_tumbleweed/gocd-agent-release-tools:latest",
        ],
        check=True,
    )
```

Comment (coolo) [reply gocd/admin/cleanup.py:127]: Tell me you don't love variables without telling me you don't love variables :)

Comment (Vogtinator) [reply gocd/admin/cleanup.py:139]: I added a variable to please all variable aficionados

**Comment by coolo** [comment]
Now we need a 3rd to approve the review :)

**Pull request merged**